src/shared/hooks/useAnalytics.ts

```diff
@@ -1,33 +1,25 @@
 import { useCallback, useState } from 'react'
-import PiwikTracker from '@amsterdam/piwik-tracker'
-import { UserOptions } from '@amsterdam/piwik-tracker/lib/types'
-import { usePiwik } from '@amsterdam/piwik-tracker-react'
+import PiwikPro, { PageViews } from '@piwikpro/react-piwik-pro'
 
-const rootUrl = process.env.REACT_APP_SELF_ROOT
-const siteId = process.env.REACT_APP_PIWIK_SITE_ID ?? ''
-const hasSiteId = !!process.env.REACT_APP_PIWIK_SITE_ID
+const dapUrl = 'https://dap.amsterdam.nl'
+const hasPiwikSiteId = !!process.env.REACT_APP_PIWIK_SITE_ID
+const piwikSiteId = process.env.REACT_APP_PIWIK_SITE_ID ?? ''
 
-let PiwikInstance: PiwikTracker
+let PiwikInstance = false
 
-const PiwikTrackerConfig: UserOptions = {
-  urlBase: rootUrl,
-  siteId,
+function createPiwikInstance(isEnabled: boolean = true) {
+  if (isEnabled && hasPiwikSiteId && dapUrl && !PiwikInstance) {
```

Comment:
If we enable analytics but don't have the necessary details like `piwikiSiteId` should we allow that? Should we not error or notify devs?

Reply:
Yeah, good point. I'll change this.

Which reminds me that this site does not have monitoring such as Sentry. Would be a good thing to add.